src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAO.scala

```diff
@@ -11,7 +11,7 @@ import scala.language.implicitConversions
 
 class GraphWorkspaceDAO extends WorkspaceDAO with GraphDAO {
 
-  override def save(workspace: Workspace, txn: RawlsTransaction): Workspace = txn withGraph { db =>
+  override def save(workspace: Workspace, txn: RawlsTransaction): WorkspaceContext = txn withGraph { db =>
```

Comment:
ah.  you're right about the return types.
disagree with your 2nd comment.  we should have hidden the type of Vertex, making it an abstract handle that only the DAO knows about.  but that was a problem before this change.  only the DAO interprets the Vertex.  and the bucket concept was already part of WorkspaceContext (it's just inside the Workspace case class now).  so that hasn't gotten any better or worse, either.